nodogsplash2/files/nodogsplash.init

```diff
@@ -248,9 +248,23 @@ create_instance() {
   procd_close_instance
 }
 
+depends() {
+  if [ "$1" = "iptables" ] ; then
+  if $WD_DIR/ndsctl status > /dev/null; then
```

Comment:
some indentation would be nice here

Reply:
@mwarning 
I made it look like the rest of the script :-D
I'll put my indentation back in.